doc/policy/packages.md

```diff
@@ -57,3 +57,40 @@ test accepts):
 
    - Warning: Batched fee-bumping may be unsafe for some use cases. Users and application developers
      should take caution if utilizing multi-parent packages.
+
+### Package Fees and Feerate
+
+*Package Feerate* is the total modified fees divided by the total virtual size of all transactions
```

Comment:
This is okay, but when I first read this, I had no idea what "modified" meant (only learning later that the `prioritisetransaction` RPC does the modifying); perhaps "modified" can be dropped? I think it's implicit. Or maybe briefly mention the RPC here.

Reply:
Added explanation